data/controls.xml

```diff
@@ -305,7 +306,7 @@
         <label fallback="label">label</label>
     </control>
     <control type="rss" id="int">
-        <width>int</width>
+        <width min="int" max="int">int</width>
```

Comment:
@phil65 Been awhile since I did this but I use it for when the play disc buttons appear for the [vertical menu option](https://github.com/mikesilvo164/Aeon-Nox-SiLVO/blob/master/16x9/IncludesHomeWidgets.xml#L710) when there is a disc inserted. That would be the easiest way I can think of to see it working. I have it in a grouplist control.